client/me/purchases/manage-purchase/index.jsx

```diff
@@ -516,6 +517,26 @@ class ManagePurchase extends Component {
 				'Transfers an existing domain from another provider to WordPress.com, ' +
 					'helping you manage your site and domain in one place.'
 			);
+		} else if ( isGSuiteOrGoogleWorkspace( purchase ) ) {
+			description = translate( 'Account for %(domain)s', {
```

Comment:
I'm wondering whether "Account for example.com" can be confusing.

Reply:
I didn't want to use `Email` since Google Workspace is more than just emails. On the other hand, we market it mainly as an email solution: I'll use the same label than for Titan.

Reply:
Actually, I will add a real description reusing the same copy that we have for emails.